--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -30,6 +30,8 @@ dtb-$(CONFIG_ARCH_QCOM) += ipq8074-ac01.dtb
 dtb-$(CONFIG_ARCH_QCOM) += ipq8074-ac02.dtb
 dtb-$(CONFIG_ARCH_QCOM) += ipq8074-ac03.dtb
 dtb-$(CONFIG_ARCH_QCOM) += ipq8074-ac04.dtb
+dtb-$(CONFIG_ARCH_QCOM) += ipq8074-ax3600.dtb
+dtb-$(CONFIG_ARCH_QCOM) += ipq8074-ax6.dtb
 dtb-$(CONFIG_ARCH_QCOM) += ipq8074-hk12.dtb
 dtb-$(CONFIG_ARCH_QCOM) += ipq9574-emulation.dtb
 dtb-$(CONFIG_ARCH_QCOM) += ipq9574-emulation-fbc.dtb
